tests/behat/theme_boost_union_feelsettings_blocks.feature

```diff
@@ -0,0 +1,227 @@
+    And I navigate to "Appearance > Default Dashboard page" in site administration
+    And I turn editing mode on
+    Then ".block-region-headertop#headertopregion" "css_element" should not exist
+    And I am on site homepage
+    And I should see "Add a block" in the ".block-region-headertop#headertopregion" "css_element"
+    Then I click on "Add a block" "link" in the ".block-region-headertop#headertopregion" "css_element"
+    And I should see "Calendar" in the ".modal-body" "css_element"
+    Then I click on "Calendar" "link" in the ".modal-body" "css_element"
+    Then I should see "Calendar" in the ".block-region-headertop#headertopregion" "css_element"
+    And I am on "Course 1" course homepage
+    And I should see "Add a block" in the ".block-region-headertop#headertopregion" "css_element"
+    Then I click on "Add a block" "link" in the ".block-region-headertop#headertopregion" "css_element"
+    Then I should see "Activities"
+    Then I click on "Activities" "link"
+    Then I should see "Activities" in the ".block-region-headertop#headertopregion" "css_element"
+    And the following "permission overrides" exist:
+      | capability                            | permission | role    | contextlevel | reference |
+      | theme/boost_union:editregionheadertop | Allow      | user    | System       |           |
+    And I log in as "student1"
+    Then I follow "Dashboard"
+    Then ".block-region-headertop#headertopregion" "css_element" should not exist
+    And I am on site homepage
+    Then I should see "Calendar" in the ".block-region-headertop#headertopregion" "css_element"
+    And I am on "Course 1" course homepage
+    Then I should see "Activities" in the ".block-region-headertop#headertopregion" "css_element"
+    And the following "permission overrides" exist:
+      | capability                            | permission | role    | contextlevel | reference |
+      | theme/boost_union:viewregionheadertop | Prevent    | user    | System       |           |
+    And I log in as "student1"
+    Then I follow "Dashboard"
+    Then I should not see "Online users"
+    Then ".block-region-headertop#headertopregion" "css_element" should not exist
+    And I am on site homepage
+    Then I should not see "Calendar"
+    Then ".block-region-headertop#headertopregion" "css_element" should not exist
+    And I am on "Course 1" course homepage
+    Then I should not see "Activities"
+    Then ".block-region-headertop#headertopregion" "css_element" should not exist
+
+  @javascript
+  Scenario: Additional Boost Union block region: offcanvas (Compared to the outside and footer regions, this test is slightly different and thus not included in the Scenario outline above).
+    When I log in as "admin"
+    And I turn editing mode on
+    And I navigate to "Appearance > Default Dashboard page" in site administration
+    And I turn editing mode on
+    #And I click on "#usernavigation .drawer-offcanvas-toggle" "css_element"
```

Comment:
This valid step is commented out which is not acceptable. Either it is needed (and should be uncommented then) or it is not needed and should be removed.

This change request applies to all subsequent commented steps in this scenario.